client/checkout/classic/upe.js

```diff
@@ -379,12 +381,13 @@ jQuery( function ( $ ) {
 		}
 		if ( ! isUPEComplete ) {
 			// If UPE fields are not filled, confirm payment to trigger validation errors
-			const { error } = await api.getStripe().confirmPayment( {
-				elements,
-				confirmParams: {
+			const { error } = await api.handlePaymentConfirmation(
+				{
 					return_url: returnUrl,
 				},
-			} );
+				elements,
+				null
```

Comment:
Should it be `paymentIntentClientSecret` here?

Reply:
Nope. The error is already known and this line only triggers validation errors like it was previously mentioned in the comment above this line: `If UPE fields are not filled, confirm payment to trigger validation errors`. I could pass `paymentIntentClientSecret`, but it is not needed anyway so I decided to skip it to prevent any strange cases.

Reply:
Tried it myself, and you are correct, we are expecting the payment confirmation to fail in that case (thus triggering the error messages below the incorrect payment fields), so the client secret is not needed